options/locale/locale_en-US.ini

```diff
@@ -1062,6 +1062,7 @@ watchers = Watchers
 stargazers = Stargazers
 stars_remove_warning = This will remove all stars from this repository.
 forks = Forks
+stars = Stars
```

Comment:
Actually if we create new keys, we may as well move them to top-level where they belong e.g. `stars` instead of `repo.stars`.

Reply:
Thought I guess for consistency we would also have to move the other keys. I guess I would grumpily accept the current key.

Reply:
And TBH I do not think "forks" and "stars" are good enough to be moved to top-level.

1. I strongly disagree to use single word key/variable name
2. "forks" are for having food, "stars" are the light points in the sky

Reply:
Likely we will never use those other meanings in Gitea context, but yeah let's keep the `repo.` prefix.